### PR TITLE
Changed %nationstatus% register type

### LINKED ISSUE #61: Change %nationstatus% to a Parentheses-type replacement.
**Is your feature request related to a problem? Please describe.**
When using %nationstatus%, if you surround it with parentheses (), it does not remove them when nation-less. This behavior, which is implemented for %nation%, would make sense to work with the status version as well.

**Describe the solution you'd like**
Converting of %nationstatus% from standard register() to registerParenthesesReplacement()

**Describe alternatives you've considered**
None (this is a simple edit)

**Additional context**
I've already made a forked version for my own testing purposes, if this is approved I can just make a pull request.

### PR DESCRIPTION
**Pull request by NinjaMandalorian**
Closes #61

### CODE REVIEW

File: maptowny-plugin/src/main/java/me/silverwolfg11/maptowny/managers/TownInfoManager.java [hunk 1]
```diff
@@ -145,6 +138,17 @@ private void registerReplacements() {
                 t-> t.hasNation() ? TownyAPI.getInstance().getTownNationOrNull(t).getName() : ""
         );
 
+        registerParenthesesReplacement("nationstatus",
```
Comment (silverwolfg11) [inline maptowny-plugin/src/main/java/me/silverwolfg11/maptowny/managers/TownInfoManager.java:141]: minor comment: looks like the new code has slight formatting differences; would prefer if it matched the formatting of the old code, but no biggie

**Review by silverwolfg11 — APPROVED** [review]
Looks good, thanks for making the change!

Did you have a chance to test it on a server?

**Comment by NinjaMandalorian** [comment]
> Looks good, thanks for making the change!
> 
> Did you have a chance to test it on a server?

Yes, I tested it on localhost and it worked as intended 👍

**Comment by NinjaMandalorian** [comment]
just saw the comment, I will return to the verbose version of the nation status ternary operator

**Comment by NinjaMandalorian** [comment]
Edit compiles and functions as intended

**Comment by silverwolfg11** [comment]
Sorry for the delay, but thanks for the work on this!

**Pull request merged**